src/Services/AssetExt.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Lykke.Service.Assets.Client.Custom;
+
+namespace Services
+{
+    /// <summary>
+    /// Extends IAsset.
+    /// IMPORTANT: necessary to perform a revision of this class ufter updating dependency
+    /// Lykke.Service.Assets.Client.
+    /// </summary>
+    public static class AssetExt
+    {
+        public static double Multiplier(this IAsset asset)
```

Comment:
Looks like it's InvertedMultiplier, since you inverts MultiplierPower sign

Reply:
As I understood (but I may be wrong), this is a capable naming. Just like in this [example|
https://bitbucket.org/lykkewalletteam/lykkewallet/src/758e1b453fa2c44e14da92ab4207fa2551b4b982/src/Core/BitCoin/Ninja/SrvNinjaBlockChainReader.cs?at=master&fileviewer=file-view-default#SrvNinjaBlockChainReader.cs-227].